modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -2959,7 +3025,7 @@ private void stopBuildIndexes() {
      * @param commitTimestamp Commit timestamp.
      */
     private void markFinished(UUID txId, TxState txState, @Nullable HybridTimestamp commitTimestamp) {
-        assert txState == COMMITED || txState == ABORTED : "Unexpected state, txId=" + txId + ", txState=" + txState;
+        assert txState.isTerminal() : "Unexpected state, txId=" + txId + ", txState=" + txState;
```

Comment:
"Unexpected state [txId=" + txId + ", txState=" + txState + ']'
More fitting to our logging format.